components/lsp/SwitchWalletLsp.styled.tsx

```diff
@@ -0,0 +1,44 @@
+import styled from "@emotion/styled";
+import { BaseButton } from "..";
+
+export const Disclaimer = styled.p`
+  margin: 0 0 26px;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 22px;
+  text-align: center;
+  color: #272528;
+`;
+
+export const SwitchButton = styled(BaseButton)`
+  width: 100%;
+  max-width: 370px;
+  margin: 0 auto;
+  padding: 0.75rem 1rem;
```

Comment:
Is this is using `rem` because we want the padding to scale with the user font-size? If there is no specific reason for using `rems` it would be nice to use `px` just to keep consistency with other parts of the app

Reply:
rem usage is intended here, yes